cmd/config/internal/commands/cmdset_test.go

```diff
@@ -129,10 +129,10 @@ apiVersion: v1alpha1
 kind: Example
 openAPI:
   definitions:
-    io.k8s.cli.setters.image:
+    io.k8s.cli.setters.image-setter:
```

Comment:
This change doesn't influence the behavior right? So we are not introducing a requirement that setters must have a `-setter` suffix.

Reply:
No. It is just a name change for tests readability.